src/app/core/components/content-top-bar/content-top-bar.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'alg-content-top-bar',
+  templateUrl: './content-top-bar.component.html',
+  styleUrls: [ './content-top-bar.component.scss' ],
+})
+export class ContentTopBarComponent {
+  @Input() fullFrameContent?: boolean;
+  @Input() currentMode?: string;
+  @Input() currentContent?: any;
```

Comment:
I think the `currentContent` is known and can be found in current content service, it shouldn't be `any` here

Reply:
Fixed